pkg/cloudprovider/providers/aws/volumes.go

```diff
@@ -63,9 +66,7 @@ func mapVolumeSpecToAwsID(name KubernetesVolumeID) (awsVolumeID, error) {
 	}
 
 	awsID := url.Path
-	if len(awsID) > 1 && awsID[0] == '/' {
-		awsID = awsID[1:]
-	}
+	awsID = strings.Trim(awsID, "/")
 
 	// TODO: Regex match?
 	if strings.Contains(awsID, "/") || !strings.HasPrefix(awsID, "vol-") {
```

Comment:
This is a "looks like a volume" sanity check.   I don't think it really makes sense to pull it out into a constant ... it isn't a constant we should be sharing IMO.